test/command/suite/sharding/logical_count/log/cover_type_all.expected

```diff
@@ -8,10 +8,15 @@ table_create Times_20150709 TABLE_PAT_KEY Time
 [[0,0.0,0.0],true]
 column_create Times_20150709 timestamp_index COLUMN_INDEX Logs_20150709 timestamp
 [[0,0.0,0.0],true]
+load --table Logs_20150709
+[
+{"timestamp": "2015-02-05 13:49:00"}
```

Comment:
This table should have only `2015-07-09` timestamps.

Reply:
Right, fix to use 2015-07-09 timestamps.